src/Compilers/CSharp/Test/Semantic/Semantics/TopLevelStatementsTests.cs

```diff
@@ -7334,6 +7334,32 @@ public void Return_01()
             Assert.True(entryPoint.ReturnsVoid);
             AssertEntryPointParameter(entryPoint);
             CompileAndVerify(comp, expectedOutput: "Return_01", args: new[] { "Return_01" });
+
+            if (ExecutionConditionUtil.IsWindows)
+            {
+                _ = ConditionalSkipReason.NativePdbRequiresDesktop;
```

Comment:
What does this statement do? Are there any side-effects?

Reply:
> What does this statement do? Are there any side-effects?

I copied the way PDBs are validated from other tests and adjusted expected base-lines. If I remember correctly that were tests targeting "Async Main" feature. I have no idea if this is doing something important or not and do not believe it is worth spending time trying to figure this out. If you are certain there is a real problem here and have a suggestion, I'll be happy to follow it.

---
In reply to: [426757323](https://github.com/dotnet/roslyn/pull/44305#discussion_r426757323) [](ancestors = 426757323)

Reply:
This seems to be a way to denote "why this block of the test is Windows-only".